src/main/java/jb/UpdateWithAdditionalOptions.java

```diff
@@ -3,15 +3,82 @@
 import jb.configuration.Configuration;
 
 import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+
+import com.beust.jcommander.IStringConverter;
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
 
 public class UpdateWithAdditionalOptions {
-    public static void main(String[] args) throws IOException {
+
+    private static class ExcludesConverter implements IStringConverter<Predicate<Path>> {
+        @Override
+        public Predicate<Path> convert(String value) {
+            return path -> FileSystems.getDefault().getPathMatcher("glob:" + value).matches(path);
+        }
+    }
+
+    @Parameter(description = "directory...")
+    private List<String> parameters = new ArrayList<>();
+
+    @Parameter(names = { "-h", "--help" }, help = true, description = "Show this help message")
+    private boolean help = false;
+
+    @Parameter(names = { "-n", "--dryrun" }, description = "Do not change files, just report what would happen")
+    private boolean dryRun = false;
```

Comment:
```suggestion
    private boolean dryRun = true;
```
Just a proposal, I made some bad experiences with command line tools that where destructive by default.